### DISCUSSION TITLE
Allure testops cloud instance returns 500 while trying to upload results

### DISCUSSION THREAD

**Discussion by YevhS** [allure-framework/allure2#2331] [TestOps]
I have 5 builds with approximately 900 integration tests in each that run in Git Hub actions CI. I use allure-mocha reporter version 2.11.1 and test ops cloud instance. And sometimes 1-2 builds fail with allurecltl upload command error `Cannot create session with request` , and I can also see 500 response :

<img width="1074" alt="image" src="https://github.com/allure-framework/allure2/assets/93822605/e81685b8-b012-4550-bf20-abb4dd691c9a">

An average successful test upload looks like this one: 

<img width="843" alt="image" src="https://github.com/allure-framework/allure2/assets/93822605/536e5c32-5227-4a23-be13-84d9fcf009b5">

There are also other builds with UI tests and the overall number of all test cases in launch is about 6000. All other builds work ok, the problem occurs only with API builds, and my first guess was that maybe it can be connected with number of test cases, because all other builds have about 200 test cases each.

So can the issue be really connected with the number of test cases in failing builds or there are some other possible issues, and how is it possible to debug it, because all I can actually see it's only 500 error.

**Comment by cheshi-mantu** [comment]
@YevhS Hi Yenhen,
please use our dedicated [help desk](https://help.qameta.io) for this type of issues. We'll be happy to help. We'll close this discussion and continue in scope of a support request as the troubleshooting might require sharing the information which should not be in public channels.

cheers
George
